mycroft/skills/npr_news/test/intent/sample2.intent.json

```diff
@@ -1,7 +1,7 @@
 {
-  "utterance": "play the news",
+  "utterance": "is there any news",
```

Comment:
The play skill of implemented media skills in the last PR ate the "play the news" utterence. To make sure the test wasn't broken @ethanaward suggested that I'd change the test.